moveit_ros/moveit_servo/tests/config/panda_simulated_config.yaml

```diff
@@ -29,13 +30,19 @@ publish_joint_accelerations: false
 ## Plugins for smoothing outgoing commands
 smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
 
+# If is_primary_planning_scene_monitor is set to true, the Servo server's PlanningScene advertises the /get_planning_scene service,
+# which other nodes can use as a source for information about the planning environment.
+# NOTE: If a different node in your system is responsible for the "primary" planning scene instance (e.g. the MoveGroup node),
+# then is_primary_planning_scene_monitor needs to be set to false.
+is_primary_planning_scene_monitor: true
+
 ## MoveIt properties
 move_group_name:  panda_arm  # Often 'manipulator' or 'arm'
 planning_frame: panda_link0  # The MoveIt planning frame. Often 'base_link' or 'world'
 
 ## Other frames
-ee_frame_name: panda_hand  # The name of the end effector link, used to return the EE pose
-robot_link_command_frame:  panda_hand  # commands must be given in the frame of a robot link. Usually either the base or end effector
+ee_frame_name: panda_link8  # The name of the end effector link, used to return the EE pose
+robot_link_command_frame:  panda_link0  # commands must be given in the frame of a robot link. Usually either the base or end effector
```

Comment:
One thing to note here is , the new implementation does not use the `robot_link_command_frame` parameter since the command frame is always included with the command.